cfgrammar/src/lib/yacc/grammar.rs

```diff
@@ -351,6 +353,11 @@ where
         &self.programs
     }
 
+    /// Get the action return type as defined by the user
+    pub fn usertype(&self) -> &Option<String> {
```

Comment:
Shouldn't this be called `actiontype` (and the attribute name too)?

Reply:
Oh, I missed that. Fixing.